pkg/sink/pulsar/factory_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pulsar
+
+import (
+	"testing"
+
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCreatorFactory(t *testing.T) {
+	client, err := NewMockCreatorFactory(nil, model.DefaultChangeFeedID("test"))
+	require.NoError(t, err)
+	require.Nil(t, client)
```

Comment:
if no error, a client should be returned ?

Reply:
We can not get a mock pulsar client,because no real pulsar server.